src/components/containers/sharing/ShareIcons.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import {
+  FacebookShareButton,
+  EmailShareButton,
+  TwitterShareButton,
+  FacebookIcon,
+  TwitterIcon,
+  EmailIcon
+} from "react-share";
+
+class ShareIcons extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
+
+  render() {
+    const template = "Highlights from Author's Haven";
+    const { shareURL, title, author } = this.props;
+    const shareTitle = `Take a look at this:  ${title}`;
+    const emailBody = `Hello\nTake a look at this article by ${author.charAt(0).toUpperCase() + author.substr(1).toLowerCase()} \n ${shareURL}`;
```

Comment:
This line is too long, kindly check and split

Reply:
Thanks for the feedback, I have refactored